packages/javascript/bh-shared-ui/src/graphSchema.ts

```diff
@@ -297,7 +281,6 @@ export const EdgeCompositionRelationships = [
     'ADCSESC9b',
     'ADCSESC10a',
     'ADCSESC10b',
-    'ADCSESC13',
```

Comment:
Is this deletion intentional? I believe main has the update to ad.cue so that this doesn't get lost when running schemagen